test/token.test.js

```diff
@@ -309,27 +287,16 @@ describe('Token', () => {
     });
 
     it('should fail sending Circles when data error', async () => {
-      // Update the edges.json file simulating data error:
+      // Update the edges.csv file simulating data error:
       // Direct path does not exist between safeAddress 0 and 4,
       // thus we create a false edge between safeAddress 0 and 4
       await Promise.resolve().then(() => {
-        const edgesData = JSON.parse(
-          execSync(
-            `docker exec circles-api cat edges-data/edges.json`,
-          ).toString(),
-        );
-        edgesData.push({
-          from: safeAddresses[0],
-          to: safeAddresses[4],
-          token: safeAddresses[0],
-          capacity: '100',
-        });
-        // Add backslashes to scape the double quote symbol
-        const edgesDataString = stringify(edgesData).replace(/"/g, '\\"');
+        let edgesCSVdata = `${safeAddresses[0]},${safeAddresses[4]},${safeAddresses[0]},100000000000000000000`;
```

Comment:
I feel I had implemented the test incorrectly before... mmmm 🤔

Reply:
can you check @JacqueGM that the test makes sense?

Reply:
test pass on my local now ^^